quickstart/examples/enrichment/README.md

```diff
@@ -0,0 +1,73 @@
+## Enrichment using Post Processors
+External Post Processor is the one that connects to an external data source to fetch data in an async manner and perform enrichment of the stream message. 
+
+In this example we use Elasticsearch as the external source.
+This allows you to enrich your data stream with the data on any remote [Elasticsearch](https://www.elastic.co/). For example, let's say you have payment transaction logs in the input stream but user profile information in Elasticsearch, then you can use this post processor to get the profile information in each record.
+
+#### Sample Configuration
+
+```properties
+{
+"external_source": {
+"es": [
+{
+"capacity": "10",
+"connect_timeout": "5000",
+"endpoint_pattern": "/customers/_doc/%s",
+"endpoint_variables": "customer_id",
+"host": "localhost",
+"output_mapping": {
+"customer_profile": {
+"path": "$._source"
+}
+},
+"port": "9200",
+"retry_timeout": "5000",
+"socket_timeout": "6000",
+"stream_timeout": "5000",
+"type": "io.odpf.dagger.consumer.EnrichedBookingLogMessage"
+}
+]
+},
+"internal_source": [
+{
+"output_field": "booking_log",
+"type": "sql",
+"value": "*"
+},
+{
+"output_field": "event_timestamp",
+"type": "function",
+"value": "CURRENT_TIMESTAMP"
+}
+]
```

Comment:
Same as this
https://github.com/odpf/dagger/pull/214/files#r1049652605

I can recommend not explaining the configs but just guiding the user through the example.